hard-protocol-home.js

```diff
@@ -85,6 +85,15 @@ const getRewardPerYearByDenom = async (siteData) => {
   return tokensDistributedBySuppliedAssetPerYear;
 }
 
+const getSupplyApyPerYearByDenom = async (siteData) => {
```

Comment:
I would rename this to `mapHardSupplyInterestRates` then instead of just `interestRate['denom']` we should format that denom so it has the `-a` at the end so it would look like `interestRate[commonDenomMapper(denom)]` = interestRate['supply_interest_rate'];

then see the comment below